pages/documents/DeveloperTools/FaaS/Developing/whitelisting.md

```diff
@@ -22,3 +22,22 @@ Please make sure to whitelist the **fully qualified domain name**. E.g. If you v
 Generally, we will prevent the double whitelisting of a domain. The UI will also indicate that this occurred using a dedicated error message. We also perform validation on the provided domain in order to ensure it is a valid domain name. For now, we do not support the whitelisting of subdomains such as **.example.com**.
 
 **Please be aware** that it might take up to **5 minutes** until the whitelisted domain becomes active on the proxy.
+
+
+### Domains Whitelisted by Default
```

Comment:
Add a link to here https://developers.liveperson.com/essential-resources-domain-api.html, so that accounts can look up the domains.

Reply:
implemented